contracts/stableswap/Swap.sol

```diff
@@ -481,10 +497,13 @@ contract Swap is OwnerPausableUpgradeable, ReentrancyGuardUpgradeable {
     /*** ADMIN FUNCTIONS ***/
 
     /**
-     * @notice Withdraw all admin fees to the contract owner
+     * @notice Update the fee address.
+     * @param newFeeAddress new fee address to be applied for fee withdrawal
      */
-    function withdrawAdminFees() external onlyOwner {
-        swapStorage.withdrawAdminFees(owner());
+    function changeFeeAddress(address newFeeAddress) external onlyOwner {
```

Comment:
Only owner can change this variable
Check if variable is changed
Check if the event is emitted